main.py

```
@@ -6,37 +6,29 @@
+from lib import analyze, pcweb
 
-    Args:
-        event (dict):  The dictionary with data specific to this type of event.
-                       The `data` field contains a description of the event in
-                       the Cloud Storage `object` format described here:
-                       https://cloud.google.com/storage/docs/json_api/v1/objects#resource
-        context (google.cloud.functions.Context): Metadata of triggering event.
-    Returns:
-        None; the output is written to Stackdriver Logging
-    """
+import os  # TODO delete
```

Pylint (3.3.7)

[warning] TODO delete

(W0511)